src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -14,15 +14,19 @@ export type ThemeProviderProps = {
    * Optional children.
    */
   children?: React.ReactNode;
+  /**
+   * Optional document specification to support popup windows.
+   */
```

Comment:
The comment could clarify the purpose and specify the default.

```suggestion
   * Document to which the theme will be applied.
   * Can be specified to handle popup windows.
   * @default document
   */
```

Reply:
Made suggested change.